Gems/ROS2/Code/Include/ROS2/Clock/SimulationClock.h

```diff
@@ -10,12 +10,16 @@
 #include <builtin_interfaces/msg/time.hpp>
 #include <rclcpp/publisher.hpp>
 #include <rosgraph_msgs/msg/clock.hpp>
+#include <AzCore/std/containers/deque.h>
+#include <AzCore/std/containers/set.h>
```

Comment:
The set does not seem to be used in this header

Reply:
Nice catch. Removed.